### PR TITLE
Spelling corrections.

### PR DESCRIPTION
**Pull request by schlessera**
Miscellaneous spelling corrections.

### CODE REVIEW

**Comment by schlessera** [comment]
Travis failed due to a SSL handshake error when connecting to SVN on its 5.2 environment.

**Comment by BE-Webdesign** [comment]
I restarted the build. Everything was passing anyways.

**Comment by BE-Webdesign** [comment]
@schlessera The JS changes need to happen on the JS repo here which are then built and merged into this repo.  Can you undo the changes in the wp-api.js so we can merge this?

File: wp-api.js [hunk 1]
```diff
@@ -179,7 +179,7 @@
 				// Add any non empty args, merging them into the args object.
```
Comment (BE-Webdesign) [inline wp-api.js:179]: We shouldn't change the JS files here, they should be changed in the [JS repo, found here](https://github.com/WP-API/client-js).  We need to remove these from here for now and when they are put into the JS repo we can build them and remerge them into this repo.

**Comment by schlessera** [comment]
@BE-Webdesign I've reset the PR to only include the typo corrections to the PHP files.

**Review by BE-Webdesign — APPROVED** [review]

**Pull request merged**